lib/Entity/Video.php

```diff
@@ -400,10 +400,14 @@ public function getStills()
         //sorting preview's images from smallest to biggest
         usort($this->stills, function (array $item1, array $item2) {
             if (isset($item1['dimension']['height'], $item2['dimension']['height'])) {
-                return $item1['dimension']['height'] > $item2['dimension']['height'];
+                if ($item1['dimension']['height'] == $item2['dimension']['height']) {
```

Comment:
Can you simplify this into a single if statement, so that you don't have two `return 0;` statements?

Reply:
@kenloburn I just fallows documentation from php docs: "The comparison function must return an integer less than, equal to, or greater than zero if the first argument is considered to be respectively less than, equal to, or greater than the second." so I need return 0 when it equal and also I need return some integer when isset return false.